src/components/ebay-notice/index.js

```diff
@@ -0,0 +1,50 @@
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+
```

Comment:
Remove this extra line. 😁

Reply:
As a side note, we may want to enforce this through eslint:
https://eslint.org/docs/rules/no-multiple-empty-lines

Reply:
I would approve of that. 👍

Reply:
oops!